tests/expected/c/10081-infinite-loops.c

```diff
@@ -0,0 +1,39 @@
+int main(void)
+{
+	int x;
+
+	for (;;)
```

Comment:
I'm expecting the user to set `sp_before_sparen`, `sp_inside_sparen`, and `sp_inside_for` in addition to `mod_infinite_loop`. Otherwise, there is no guarantee that loops will have similar spacing. Do you want me to set `sp_before_sparen` in the tests so that all the loops in the expected output look the same?

Reply:
Ah, I see now. The ones with the additional space are those cases which were not replaced and so are the same as the original code. In such case I think it should be fine, because other options should take care of normalizing spacing later on. I have not tested, but if you have verified that doing so is enough, then we can just ignore my original comment.